dev/almalinux-8/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf -y install \
         hunspell-en \
         kdesdk-kcachegrind \
         graphviz && \
-    pip3 install xgboost scikit-learn && \
+    pip3 install numpy xgboost scikit-learn && \
```

Comment:
I believe you need `pip3 install --upgrade pip` before installing `xgboost` and `scikit-learn`.

Reply:
Thanks, seems to work now!